### PR TITLE
Add links to the install- and dev-manuals

### PR DESCRIPTION
**Pull request by cdingm**

### CODE REVIEW

File: profile/README.md [hunk 1]
```diff
@@ -10,6 +10,14 @@ They determine what questions the students are asked in a minigame, what hints t
 
 For an in depth explanation of each service go to the **[user manuals](https://gamifyit-docs.readthedocs.io/en/latest/user-manuals/index.html)**
 
+## Installation Manuals
+
+For informations about the configuration of services go to the **[installation manuals](https://gamifyit-docs.readthedocs.io/en/latest/install-manuals/index.html)**
```
Comment (delvh) [inline profile/README.md]:
```suggestion
To learn how to install everything/ specific services, visit the **[installation manuals](https://gamifyit-docs.readthedocs.io/en/latest/install-manuals/index.html)**.
```

File: profile/README.md [hunk 2]
```diff
@@ -10,6 +10,14 @@ They determine what questions the students are asked in a minigame, what hints t
...
+
+## Developer Manuals
+
+Documents for Developers with a lot of knowledge about the projects can be found in the **[developer manuals](https://gamifyit-docs.readthedocs.io/en/latest/dev-manuals/index.html)**
```
Comment (delvh) [inline profile/README.md]:
```suggestion
Specific knowledge about services/ the architecture can be found in the **[developer manuals](https://gamifyit-docs.readthedocs.io/en/latest/dev-manuals/index.html)**.
```

**Comment by madebyTimo** [comment]
I commited my changes for the run-config directly into this branch to prevent merge conflicts. I hope this is ok

File: profile/README.md [hunk 3]
```diff
@@ -12,7 +12,7 @@ For an in depth explanation of each service go to the **[user manuals](https://g
 
 ## Installation Manuals
 
-For informations about the configuration of services go to the **[installation manuals](https://gamifyit-docs.readthedocs.io/en/latest/install-manuals/index.html)**
+To learn how to install everything, visit the **[installation manuals](https://gamifyit-docs.readthedocs.io/en/latest/install-manuals/index.html)** For a template on how to host checkout the [run config](https://github.com/Gamify-IT/run-config).
```
Comment (delvh) [inline profile/README.md]:
```suggestion
To learn how to install everything, visit the **[installation manuals](https://gamifyit-docs.readthedocs.io/en/latest/install-manuals/index.html)**. For a template on how to host checkout the [run config](https://github.com/Gamify-IT/run-config).
```

**Pull request merged**